### PR TITLE
add integration test for getCarbonOffsetCosts calculation.

### PR DESCRIPTION
**Pull request by ahmetoz**

### CODE REVIEW

File: extension/test/integration/get-carbon-offset-costs.handler.spec.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      packageWeight: {
+        value: 2.2,
+        unit: 'kg',
+      },
+    }
+    const interfaceInteractionResponse = await getCarbonOffsetCosts(
+      getCarbonOffsetCostsRequestDraft
+    )
+    expect(interfaceInteractionResponse.deliveryOffset).to.exist
+    expect(interfaceInteractionResponse.totalOffset).to.exist
```
Comment (leungkinghin) [inline extension/test/integration/get-carbon-offset-costs.handler.spec.js:34]: why not do assertion for `deliveryOffset.value` as well? (Same in line 66-69)

File: extension/src/service/web-component-service.js [hunk 2]
```diff
@@ -94,7 +94,7 @@ function getCarbonOffsetCosts(merchantAccount, getCarbonOffsetCostsRequestObj) {
     `${adyenCredentials.apiBaseUrl}/carbonOffsetCosts`,
     merchantAccount,
     adyenCredentials.apiKey,
-    extendRequestObjWithApplicationInfo(getCarbonOffsetCostsRequestObj)
+    getCarbonOffsetCostsRequestObj
```
Comment (lojzatran) [inline extension/src/service/web-component-service.js:97]: Why did you make this change?

Comment (ahmetoz) [reply extension/src/service/web-component-service.js:97]: carbonOffsetCosts endpoint does not accept applicationInfo field 🤷🏽

File: extension/test/integration/get-carbon-offset-costs.handler.spec.js (hunk 1 above)
Comment (ahmetoz) [reply extension/test/integration/get-carbon-offset-costs.handler.spec.js:35]: https://github.com/commercetools/commercetools-adyen-integration/pull/857/commits/1870344d491abbb7f777e4ff7399bfbdfef41d4a

File: extension/src/service/web-component-service.js (hunk 2 above)
Comment (lojzatran) [reply extension/src/service/web-component-service.js:97]: Can you inform Adyen about this? I guess they want to have this info sent to them.

**Pull request merged**